### ISSUE TITLE
skupper link status 

### ISSUE THREAD

**Issue by pwright** [skupperproject/skupper#437]
After creating link for 'hello world' 

```
west $ skupper link status
There are no connectors configured or active

east $ skupper link status
Connection for conn1 is active
```

On west `skupper status` gives me

```
It is connected to 1 other site.
```
which is more info than link status  gives me, should `link status` provide the same info on either side of the connection?

**Comment by ted-ross** [comment]
On east, `skupper link status` shows the status of the connections for the connectors.  On west, it simply says that there are no connectors but doesn't mention incoming connections.
I think an argument can be made that link status should include incoming connections as well as outgoing.

**Comment by grs** [comment]
A counter argument is that `skupper link status` gives information on objects created by `skupper link create` which is outgoing links. Those outgoing links can be deleted with `skupper link delete`.

While I agree that providing more insight into incoming links as well is useful, I'm not convinced `skupper link status` in its current form is the right place for that. Mixing directions muddies the pattern in my view.

**Comment by nluaces** [comment]
This will be addressed  by #878 showing a difference between local links (initiated from the site) and remote links.
